packages/picasso/src/Amount/story/index.jsx

```diff
@@ -10,13 +10,15 @@ const page = PicassoBook.section('Components').createPage(
   `
 )
 
-page
-  .createTabChapter('Props')
-  .addComponentDocs({ component: Amount, name: 'Amount' })
+page.createTabChapter('Props').addComponentDocs({
+  component: Amount,
+  name: 'Amount',
+})
 
 page
   .createChapter()
   .addExample('Amount/story/Default.example.tsx', 'Default')
   .addExample('Amount/story/Currency.example.tsx', 'Currency Variants')
   .addExample('Amount/story/Variants.example.tsx', 'Typography Variants')
   .addExample('Amount/story/Locale.example.tsx', 'Locale Variants')
+  .addExample('Amount/story/Decimals.example.tsx', 'Decimal digits')
```

Comment:
```suggestion
  .addExample('Amount/story/Decimals.example.tsx', 'Fractional digits')
```

I reckon this would make more sense as a title, WDYT?